website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx

```diff
@@ -113,6 +113,10 @@ To create a policy set:
 1. Choose a policy set scope:
    - **Policies enforced globally:** Terraform Cloud automatically enforces this global policy set on all of an organization's existing and future workspaces.
    - **Policies enforced on selected projects and workspaces:** Use the text fields to find and select the workspaces and projects to enforce this policy set on. This affects all current and future workspaces for any chosen projects.
+1. **(Sentinel Only) Choose a policy set type:
+   - **Standard:** .
+   - **Policies enforced on selected projects and workspaces:** Use the text fields to find and select the workspaces and projects to enforce this policy set on. This affects all current and future workspaces for any chosen projects.
```

Comment:
@mrinalirao this line looks like it has shifted and should go above in the `choose a policy set scope` section